apps/synapse-oauth-signin/src/style/theme.ts

```diff
@@ -1,3 +1,4 @@
+import 'synapse-react-client/dist/utils/theme/ThemeTypes'
```

Comment:
Import the declared types so we can safely use our custom theme keys like typography variants